djweb/dj_comp_hist/common.py

```diff
@@ -0,0 +1,86 @@
+import os
+from pathlib import Path, PurePosixPath
+from .models import Document, Page
+from django.db.models.signals import post_save
+
+
+DJWEB_PATH = Path(os.path.abspath(os.path.dirname(__file__)))
+DATA_BASE_PATH = Path(DJWEB_PATH.parent,"computation_hist","data","processed_pdfs")
+
+
+def get_file_path(box: int, folder: int, foldername_short:str, file_type:str,
+                  doc_id:int = None, page_id:int = None, include_base_path=True, aws_file=False):
```

Comment:
Can you replace `include_base_path=True, aws_file=False` with a param `path_type` ?
The path_type can be: 
relative (currently: include_base_path=False, aws_file=False)
absolute (currently: include_base_path=True, aws_file=False)
aws (currently: include_base_path=False, aws_file=True)

Currently, I think you'll produce errors if include_base_path=True and aws_file=True. 
Plus, I think my variable name (include_base_path) isn't clear. relative and absolute are more established names.